src/modules/src/estimator_kalman.c

```diff
@@ -447,34 +447,121 @@ void estimatorKalmanGetEstimatedRot(float * rotationMatrix) {
   memcpy(rotationMatrix, coreData.R, 9*sizeof(float));
 }
 
-// Stock log groups
+/**
+ * Variables and results from the Extended Kalman Filter
```

Comment:
maybe link here? https://www.bitcraze.io/documentation/repository/crazyflie-firmware/2020.04/functional-areas/state_estimators/

Reply:
Good question if we want (how many) links back to the web docs. They may be hard to maintain over time when we update the web? 
If there is a link it should link to master though: https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/functional-areas/state_estimators/